api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/RepositoryMediator.java

```diff
@@ -149,7 +149,6 @@ public void ageDatabaseEntities(int numberOfDays) {
         paymentRepository.findAll().forEach(payment -> EntityAgeAccelerator.ageObject(payment, numberOfDays));
         claimRepository.findAll().forEach(claim -> {
             EntityAgeAccelerator.ageObject(claim, numberOfDays);
-            EntityAgeAccelerator.ageObject(claim.getClaimant(), numberOfDays);
```

Comment:
Why do we now no longer need to age the Claimant, did we even need to do it in the first place?

Reply:
We did age the claimant before, but it's now being aged via the claim. We removed this line to prevent claimants being aged twice. The EntityAgeAccelerator ages Temporal fields in the object, and all child (non-entity) objects. Claim is no longer an entity, so it is included when we age the claim.